### ISSUE TITLE
Questions about training on my own dataset (gray-scale, 1 class)

### ISSUE THREAD

**Issue by shanruiyu** [facebookresearch/maskrcnn-benchmark#1119]
## ❓ Questions and Help
Hi,
Thanks for reading my issue!

I have a dataset consisting of 1.4k thyroid ultrasound images (1.4k ultrasound images(.png) and 1.4k mask image(.png)), and they are grayscale images.  My dataset has only one category — throid (excluding background). I want to train a model based on mask-rcnn to segment thyroid nodules.

Here’s what I have done:
1. Converting data set into COCO format by using pycococreator.
2. Changing some configs.
3. Run the train_net.py and test_net.py.
The result is not very satisfactory.

Writing this issue, I reverently ask that:
1. How to change some code to train gray-scale images?
2. How to determine the _C.SOLVER.MAX_ITER and _C.SOLVER.STEPS with about 1k images?
3. I don't quite understand the evaluation indicators for coco datasets, such as AP，AP50 and AP75. Can you explain it for me?
4. Is there any way to calculate the dice coefficients of the model on the test set?
5. Is there any way to use the mask images(.png) directly to train without converting it into coco? 

Thank you very much!

**Comment by Darshan2701** [comment]
@shanruiyu Were you able to run the train_net.py?

**Comment by shanruiyu** [comment]
> @shanruiyu Were you able to run the train_net.py?
Yes!  And I can also run test_net.py,  I curious to know how to calculate the dice coefficients on coco format dataset.

**Comment by mlszy928** [comment]
I have similar problems with you,  have you solved your problems?  Can you add my QQ to have some discussion? My QQ is 304378542
